### PR TITLE
Slight performance improvments

### PR DESCRIPTION
**Pull request by theopolis**

### CODE REVIEW

**Comment by osqueryer** [comment]
Build origin/pr/2242/merge (Job: 2866) failed :-1:

**Comment by theopolis** [comment]
ok to test

**Comment by osqueryer** [comment]
Build origin/pr/2242/merge (Job: 2867) failed :-1:

**Comment by theopolis** [comment]
ok to test

**Comment by osqueryer** [comment]
Build origin/pr/2242/merge (Job: 2870) failed :-1:

File: osquery/core/windows/process.cpp [hunk 1]
```diff
@@ -99,11 +99,9 @@ std::shared_ptr<PlatformProcess> PlatformProcess::getLauncherProcess() {
   try {
     handle = reinterpret_cast<HANDLE>(static_cast<std::uintptr_t>(
         std::stoull(*launcher_handle, nullptr, 16)));
-  }
-  catch (std::invalid_argument e) {
+  } catch (std::invalid_argument e) {
```
Comment (zwass) [inline osquery/core/windows/process.cpp]: Don't we usually try to catch exceptions as `const` references?

**Pull request merged**